tests/e2e/scripts/cloud-run/deploy.sh

```diff
@@ -116,6 +118,19 @@ function deployBackend() {
 
       BACKEND_HOST=$(gcloud functions describe ${BACKEND_SERVICE_NAME}  --format="value(httpsTrigger.url)" --quiet)
       ;;
+    "app-engine")
+    cd ${ROOT}/tests/endpoints/bookstore
+
+    sed "s/SERVICE_NAME/${BACKEND_SERVICE_NAME}/g" app_template.yaml > app.yaml
+    gcloud app deploy --quiet
```

Comment:
When I try to access the app deployed by this e2e test, I get the following message: https://screenshot.googleplex.com/zLxteKdrC25

Can you check if any permissions need to be set up? Why does it have your email if the prow service account is the one deploying it?

Reply:
I enable IAP for the App Engine of XXXcloudesf-testing.uc.r.appspot.com and the service of this test will be deployed under this. You needroles/iap.httpsResourceAccessor to access them.

Nop, IAP is not deployed. Its OAuth consent screen is created by my email.

Reply:
Ah right, makes sense. Forgot you need to configure the consent screen.